### PR TITLE
fix: Dark mode and text area bug

### LINKED ISSUE #15: Dark mode and text area bug
Found a bug? Please fill out the sections below. 👍

### Issue Summary

some of the text color doesn't change according to the mode of webpage [dark/light] in form page and user can resize the size of text area which disturb the Ui. 

### Steps to Reproduce

1. went to form that I have created.
2. when I clicked the form that I have created and navigate to all the accordions I found there is some problem with text and background which color doesn't change according to the mode of the page
3. text area also resizable that might hamper the Ui of webpage when user try to resize [actually when user try to increase the size]

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

### Actual Results

- al the Ui is fine even in dark mode also

### Expected Results

- This is an ideal result that the system should get after the tests are performed

### Technical details
Screenshot 2023-10-07 182830
Screenshot 2023-10-07 182847
Screenshot 2023-10-07 183216
Screenshot 2023-10-07 183704

- Node.js version
- Anything else that you think could be an issue.

### Evidence

- How was this tested? This is quite mandatory in terms of bugs. Providing evidence of your testing with screenshots or/and videos is an amazing way to prove the bug and a troubleshooting chance to find the solution.
Screenshot 2023-10-07 184128
Screenshot 2023-10-07 184138

### PR DESCRIPTION
**Pull request by NamelesssNerd**
## What does this PR do?

Fixes #15

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: components/Dashboard/ReviewForm.tsx [hunk 1]
```diff
@@ -75,12 +75,12 @@ const ReviewForm: React.FC<ReviewFormProps> = (props) => {
         {/* Design  */}
         {currentStepId === 'WELCOME_PAGE' && (
           <div className="relative w-full max-w-lg rounded-xl bg-white p-6 pt-4 shadow-lg">
-            <div className="absolute -top-3.5 right-3 flex items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm">
-              <HeartFilledIcon className="h-4 w-4" />
-              Powered by Review
-            </div>
+              <div className="absolute -top-3.5 right-3 flex dark:text-black items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm">
```
Comment (piyushgarg-dev) [inline components/Dashboard/ReviewForm.tsx:78]: @hemantwasthere Can you please verify this?

File: components/Dashboard/ReviewForm.tsx [hunk 2]
```diff
@@ -100,12 +100,12 @@ const ReviewForm: React.FC<ReviewFormProps> = (props) => {
         {/* Response Page  */}
         {currentStepId === 'RESPONSE_PAGE' && (
           <div className="relative w-full max-w-lg rounded-xl bg-white p-6 pt-4 shadow-lg">
-            <div className="absolute -top-3.5 right-3 flex items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm">
-              <HeartFilledIcon className="h-4 w-4" />
+            <div className="absolute -top-3.5 right-3 flex items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm dark:text-black">
```
Comment (piyushgarg-dev) [inline components/Dashboard/ReviewForm.tsx:103]: @hemantwasthere This as well

**Review by piyushgarg-dev — CHANGES_REQUESTED** [review]

File: components/Dashboard/ReviewForm.tsx [hunk 3]
```diff
@@ -206,7 +207,7 @@ const ReviewForm: React.FC<ReviewFormProps> = (props) => {
                             alt="logo"
                           />
                         </div>
-                        <p className="rounded-lg border border-input bg-background px-4 py-2 font-semibold hover:bg-accent hover:text-accent-foreground">
+                        <p className="rounded-lg border border-input bg-background px-4 py-2 font-semibold hover:bg-accent hover:text-accent-foreground dark:text-white dark:bg-[#1f2937]">
```
Comment (hemantwasthere) [inline components/Dashboard/ReviewForm.tsx:210]: please remove this `dark:bg-[#1f2937]` class

File: components/Dashboard/ReviewForm.tsx [hunk 4]
```diff
@@ -159,12 +160,12 @@ const ReviewForm: React.FC<ReviewFormProps> = (props) => {
...
           <>
             <div className="relative mb-8 mt-96 w-full max-w-lg rounded-xl bg-white p-6 pt-4 shadow-lg">
-              <div className="absolute -top-3.5 right-3 flex items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm">
+              <div className="absolute -top-3.5 right-3 flex items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm dark:text-black">
                 <HeartFilledIcon className="h-4 w-4" />
                 Powered by Review
               </div>
               <div className="my-2">
-                <HeartFilledIcon className="h-12 w-12" />
+              <HeartFilledIcon className="h-12 w-12 text-black" />
```
Comment (hemantwasthere) [inline components/Dashboard/ReviewForm.tsx:168]: put an indent in front of `<HeartFilledIcon` like the before it has

File: components/Dashboard/ReviewForm.tsx [hunk 5]
```diff
@@ -100,12 +100,12 @@ const ReviewForm: React.FC<ReviewFormProps> = (props) => {
...
           <div className="relative w-full max-w-lg rounded-xl bg-white p-6 pt-4 shadow-lg">
-            <div className="absolute -top-3.5 right-3 flex items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm">
-              <HeartFilledIcon className="h-4 w-4" />
+            <div className="absolute -top-3.5 right-3 flex items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm dark:text-black">
+              <HeartFilledIcon className="h-4 w-4 " />
               Powered by Review
             </div>
             <div className="my-2 flex w-full justify-between">
-              <HeartFilledIcon className="h-12 w-12" />
+            <HeartFilledIcon className="h-12 w-12 text-black" />
```
Comment (hemantwasthere) [inline components/Dashboard/ReviewForm.tsx:108]: put an indent in front of `<HeartFilledIcon />` like the before it has

File: components/Dashboard/ReviewForm.tsx [hunk 6]
```diff
@@ -100,12 +100,12 @@ const ReviewForm: React.FC<ReviewFormProps> = (props) => {
         {/* Response Page  */}
         {currentStepId === 'RESPONSE_PAGE' && (
           <div className="relative w-full max-w-lg rounded-xl bg-white p-6 pt-4 shadow-lg">
-            <div className="absolute -top-3.5 right-3 flex items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm">
-              <HeartFilledIcon className="h-4 w-4" />
+            <div className="absolute -top-3.5 right-3 flex items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm dark:text-black">
+              <HeartFilledIcon className="h-4 w-4 " />
```
Comment (hemantwasthere) [inline components/Dashboard/ReviewForm.tsx:104]: remove this space after `w-4`

File: components/Dashboard/ReviewForm.tsx [hunk 7]
```diff
@@ -75,12 +75,12 @@ const ReviewForm: React.FC<ReviewFormProps> = (props) => {
...
         {currentStepId === 'WELCOME_PAGE' && (
           <div className="relative w-full max-w-lg rounded-xl bg-white p-6 pt-4 shadow-lg">
-            <div className="absolute -top-3.5 right-3 flex items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm">
-              <HeartFilledIcon className="h-4 w-4" />
-              Powered by Review
-            </div>
+              <div className="absolute -top-3.5 right-3 flex dark:text-black items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm">
+                <HeartFilledIcon className="h-4 w-4" />
+                Powered by Review
+              </div>
```
Comment (hemantwasthere) [inline components/Dashboard/ReviewForm.tsx:81]: remove these spaces in the front of 79,80,81 lines.

File: components/Dashboard/ReviewForm.tsx (hunk 1 above)
Comment (hemantwasthere) [inline components/Dashboard/ReviewForm.tsx:78]: remove this space in the front of 78 line

Comment (hemantwasthere) [reply components/Dashboard/ReviewForm.tsx:78]: okay

File: components/Dashboard/ReviewForm.tsx (hunk 2 above)
Comment (hemantwasthere) [reply components/Dashboard/ReviewForm.tsx:103]: 👍

File: components/Dashboard/ReviewForm.tsx [hunk 8]
```diff
@@ -271,7 +272,7 @@ const ReviewForm: React.FC<ReviewFormProps> = (props) => {
           <div className="flex w-full max-w-lg flex-col items-center justify-center">
             <div className="w-full rounded-md bg-white px-6 py-4 shadow-lg">
               <div className="my-2">
-                <HeartFilledIcon className="h-12 w-12" />
+              <HeartFilledIcon className="h-12 w-12 text-black" />
```
Comment (hemantwasthere) [inline components/Dashboard/ReviewForm.tsx:275]: add a space in front of `<HeartFilledIcon` of 275 line

**Review by hemantwasthere — CHANGES_REQUESTED** [review]
Please be aware of indentation before committing your code next time else everything looks good. 👍

File: components/Dashboard/FormPreview.tsx [hunk 9]
```diff
@@ -21,11 +21,11 @@ const FormPreview: React.FC<FormPreviewProps> = (props) => {
         'mb-10': isMobile,
       })}
     >
-      <div className="mb-4 flex justify-center rounded-lg bg-gray-300 p-1">
+      <div className="mb-4 flex justify-center rounded-lg bg-gray-300 dark:bg-gray-700 p-1">
         <button
           className={cn(
             'offset_ring rounded-md p-1.5 transition-all',
-            isMobile && 'bg-gray-100'
+            isMobile && 'bg-gray-100 dark:bg-black'
```
Comment (hemantwasthere) [inline components/Dashboard/FormPreview.tsx:28]: put `dark:bg-gray-500` here also

**Review by hemantwasthere — CHANGES_REQUESTED** [review]

File: components/Dashboard/FormPreview.tsx (hunk 9 above)
Comment (hemantwasthere) [reply components/Dashboard/FormPreview.tsx]: yup everything looks good now!

**Review by hemantwasthere — APPROVED** [review]

**Comment by hemantwasthere** [comment]
@NamelesssNerd Thanks for your meaningful contributions 😊

**Pull request merged**